testing/asserts_test.ts

```diff
@@ -213,6 +213,11 @@ Deno.test("testingAssertExists", function (): void {
   assertExists(-0);
   assertExists(0);
   assertExists(NaN);
+
+  const value = new URLSearchParams({ value: "test" }).get("value");
+  assertExists(value);
+  assertEquals(value.length, 4);
```

Comment:
@kt3k PTAL. I replaced `console.log` in the last line with `assertEquals` in order to reduce noise.

Reply:
Looks good to me. Thanks for updating!